src/Orchard/Localization/Services/LocalizationStreamParser.cs

```diff
@@ -96,27 +103,33 @@ public class LocalizationStreamParser : ILocalizationStreamParser {
             return sb == null ? str : sb.ToString();
         }
 
-        private static string TrimQuote(string str) {
```

Comment:
Because I need to access the `Logger` property (which is non-static) to write a warning to the log file. Alternatively, I can leave out the warning and just interpret this single quote as an empty string. I'm not sure which solution is better.